### ISSUE TITLE
Improving testability by standardizing `id` and `class` attributes

### ISSUE THREAD

**Issue by OR13** [digitalbazaar/chapi-interop-test-suite#1]
Seems like the vendors are currently allowed to implement features with lots of customization.

for example: https://github.com/digitalbazaar/chapi-interop-test-suite/blob/main/test/vendors/transmute/issuer.js#L32

It would be better to agree to standard `id`, `class` fields for exposing behavior.

**Comment by gannan08** [comment]
@OR13 You are correct. Vendors are allowed to customize how they interact with their application as long as they adhere to the pre-conditions and post-conditions of the API. With this being the case, then it shouldn't matter what `id` or `class` attribute is used.

There may have been some missed use cases in the initial design. Does the current design prevent a use case that you would like to achieve?

**Comment by OR13** [comment]
@gannan08 no, but it makes each vendors integration much more "custom" / less similar, and makes it harder to "copy paste" to get similar functionality... this will lead to more creativity inside the chapi window, which in turn will cut down on the users ability to leave one system and go to another... for example, android or iphone users have little trouble upgrading to the same kind of phone, but are generally lost when switching.... if CHAPI is successful without some basic class names / attributes we will likely see wildy different UIs... which certainly can be web driven... I suppose we can cross that bridge when / if  it becomes an issue.
